src/controls/QskSegmentedBar.h

```diff
@@ -48,6 +48,10 @@ class QSK_EXPORT QskSegmentedBar : public QskControl
 
     int addText( const QString& );
     int addGraphic( const QUrl& );
+    int addGraphicAndText( const QUrl&, const QString& );
+
+    QString textAt( int ) const;
+    QskGraphic graphicAt( int ) const;
```

Comment:
Since we can have both text and graphic at a certain index, I wonder whether we still need
```
QVariant optionAt( int ) const
```
We could return a QPair if both tex and graphic are set, but that seems a bit weird...

Reply:
Guess I once started with copying the API from QskMenu, where optionAt() returns a QVariantList - at least returning a QVariantList() in QskSegmentedBar::optionAt indicates this. The idea might have been to make optionAt being virtual at some point, so that customized segmented bars ( or menus ) could have more than only text and/or graphic.

As I got distracted with other tasks the API was left in an incomplete - or even messed up - state, as we can see here. In general I would like to have very similar APIs for all controls, that do these "select one of the icon/text pairs"  ( combo boxes and certain list views comes to mind as well ) - maybe even having a common base class providing this API.

Reply:
sounds good; shall I for the scope of this PR just change the API so it resembles the one from QskMenu, i.e. call the methods "void addOption()" and "QskVariantList optionAt()"?
I also wonder whether e.g. textAt() would be more convenient by just returning an empty string if not set...

Reply:
Yes please go ahead and align the APIs. textAt/graphicAt might be convenient, but should rely on the list you get from optionAt.

Reply:
ok, I changed the API so that we have `addOption()` and `optionAt()` now, just like in QskMenu. I didn't add `textAt()` or `graphicAt()` yet, since QskMenu doesn't have that either.

We could also make `qskValueAt()` that is used in QskMenuSkinlet public, but maybe this is something for a new PR then...